charts/metering-ansible-operator/templates/crds/reportdatasource.crd.yaml

```diff
@@ -39,3 +39,113 @@ spec:
   - name: Age
     type: date
     JSONPath: .metadata.creationTimestamp
+  validation:
+    openAPIV3Schema:
+      type: object
+      required:
+      - spec
+      properties:
+        spec:
+          type: object
+          properties:
+            prometheusMetricsImporter:
+              type: object
+              required:
+              - query
+              properties:
+                query:
+                  type: string
+                  minLength: 1
+                storage:
+                  type: object
+                  properties:
+                    storageLocationName:
+                      type: string
+                      minLength: 1
+                prometheusConfig:
+                  type: object
+                  required:
+                  - url
+                  properties:
+                    url:
+                      type: string
+                      format: uri
+            reportQueryView:
+              type: object
+              required:
+              - queryName
+              properties:
+                queryName:
+                  type: string
+                  minLength: 1
+                inputs:
+                  type: array
+                  minItems: 1
+                  items:
+                    type: object
+                    required:
+                    - name
+                    - type
+                    properties:
+                      name:
+                        type: string
+                        minLength: 1
+                      type:
```

Comment:
Looking again, I see how this happened cause our documentation is confusing, but there is no `type` field for this, we need `values`, like reports instead.